apps/src/aichat/views/AichatView.tsx

```diff
@@ -96,8 +97,10 @@ const AichatView: React.FunctionComponent = () => {
     projectManager.addSaveSuccessListener(() => {
       dispatch(onSaveComplete());
     });
-    projectManager.addSaveFailListener(() => {
-      dispatch(onSaveFail());
+    projectManager.addSaveFailListener((e: Error) => {
+      if (e instanceof NetworkError) {
+        dispatch(onSaveFail(e));
+      }
```

Comment:
I think it would still be good to show an error message even if we don't receive a `NetworkError` - could we pass in the error to `onSaveFail` and then check the error type there to determine what string to display?